src/main/resources/assets/music_control/lang/en_us.json

```diff
@@ -56,7 +56,16 @@
     "music.volume": "Volume: %s%%",
     "music.none": "None",
 
+    "music.category.null": "None",
+    "music.category.all": "All",
+    "music.category.default": "Default",
+    "music.category.disc": "Discs",
+
+    "music.format.category": "Playing Category: %s with Music: %s",
+    "music.format.biome": "Biome: %s",
```

Comment:
Your change is nice, but that makes me feel disc should also be displayed this way for consistency.
Why not add a `"music.format.disc": "Disc: %s",`